functions/source/rest_client.py

```diff
@@ -294,6 +299,35 @@ def create_workspaces(account_id, workspace_name, deployment_name, aws_region, c
        
             response.update(response_policy)
             print(response)
+
+            # DEFAULT CLUSTER
+            if default_cluster:
+                
+                CLUSTER_URL = "https://"+deployment_name+".cloud.databricks.com/api/2.0/clusters/create"
+
+                CLUSTER_DATA = {
+                    "cluster_name": "single-node-cluster",
+                    "spark_version": "7.6.x-scala2.12",
+                    "node_type_id": "M4.xlarge",
+                    "num_workers": 0,
+                    "start_cluster": False,
+                    "spark_conf": {
+                        "spark.databricks.cluster.profile": "singleNode",
+                        "spark.master": "local[*]"
+                    },
+                    "custom_tags": {
+                        "ResourceClass": "SingleNode"
+                    }
+                }
+
+                
+
+                default_cluster = post_request(CLUSTER_URL, CLUSTER_DATA, encodedbase64, user_agent, version)
```

Comment:
We're still going to be catching errors for this call right?

Reply:
yeah, we will going to add the error handling after this PR is reviewed.